### ISSUE TITLE
Not respecting set stop loss, instead using -100% tolerance

### ISSUE THREAD

**Issue by kaiserex** [freqtrade/freqtrade#3817]
## Describe your environment

  * Operating system: Ubuntu 20.04
  * Python Version: Python 3.8.2 
  * CCXT version: ccxt==1.33.18
  * Freqtrade Version: develop-dcedc148

## Describe the problem:

Bot is not respecting the set stop loss in the strategy. I am running a very primitive strategy based on the sample:
 ```
"""
    Default Strategy provided by freqtrade bot.
    You can override it with your own strategy
    """

    # Minimal ROI designed for the strategy
    minimal_roi = {
        "0": 0.005
    }

    # Stoploss:
    stoploss = -0.35

	# Optimal ticker interval for the strategy
    ticker_interval = '5m'

    # Optional order type mapping
    order_types = {
        'buy': 'limit',
        'sell': 'limit',
        'stoploss': 'market',
        'stoploss_on_exchange': False
    }

    # Optional time in force for orders
    order_time_in_force = {
        'buy': 'gtc',
        'sell': 'gtc',
    }

```

Corresponding part of config file:
```
 "max_open_trades": 20,
    "stake_currency": "BTC",
    "stake_amount": 0.005,
    "tradable_balance_ratio": 0.99,
    "fiat_display_currency": "USD",
    "timeframe": "5m",
    "dry_run": false,
    "cancel_open_orders_on_exit": true,
    "trailing_stop": false,
    "unfilledtimeout": {
        "buy": 15000,
        "sell": 45000
    },
    "bid_strategy": {
        "ask_last_balance": 1.0,
        "use_order_book": true,
        "order_book_top": 1,
        "check_depth_of_market": {
            "enabled": false,
            "bids_to_ask_delta": 1
        }
    },
    "ask_strategy": {
        "use_order_book": true,
        "price_side": "ask",
        "order_book_min": 1,
        "order_book_max": 10,
        "order_book_top": 1,
        "use_sell_signal": false,
        "sell_profit_only": true,
    },
```
This image shows the example of -100% stop loss on one of the pairs, when running /status command on telegram.
![Screenshot from 2020-10-01 12-19-53](https://user-images.githubusercontent.com/25899070/94793151-b5a65080-03e2-11eb-82e3-bc31d3d79969.png)
I am guessing this might be related due to low price of the coin and some calculation precision.
Looking forward to what you think?

Let me know if i need to provide more information.

Best Regards,
Linas

**Comment by xmatthias** [comment]
While the title is very catchy - it's just displayed / handled in an odd way (your stoploss is never really at 100%).

Your telegram-message shows that there is an open buy order. stoploss will only be calculated once that order is filled (before that, we don't know the final (average) price - which could be your limit price or in your favor).
Once the buy-order fills, stoploss will be updated to the correct value - and this display will be correct again.

The best option is probably to not display stoploss until the buy-order is filled (since the entry price is not clear, stoploss is neither).

**Comment by kaiserex** [comment]
Thanks for explaining, this should be closed, didn't notice the trades weren't opened yet.
